tests/api/garage/cars/getCars.spec.js

```diff
@@ -0,0 +1,104 @@
+      const response = await client.carController.getBrands();
+      expect(response.status, "Status code should be valid").toBe(200);
+      expect(response.data.data.length).toBeGreaterThan(0);
+      expect(response.data.data[0]).toHaveProperty("id");
+      expect(response.data.data[0]).toHaveProperty("logoFilename");
+    });
+
+    test("Get car brand by id", async () => {
+      const response = await client.carController.getBrandsById("1");
+      expect(response.status, "Status code should be valid").toBe(200);
+      expect(response.data.data).toHaveProperty("id");
+      expect(response.data.data).toHaveProperty("title");
+      expect(response.data.data).toHaveProperty("logoFilename");
+    });
+
+    test("Get cars models", async () => {
+      const response = await client.carController.getModels();
+      expect(response.status, "Status code should be valid").toBe(200);
+      expect(response.data.data.length).toBeGreaterThan(0);
+      expect(response.data.data[0]).toHaveProperty("id");
+      expect(response.data.data[0]).toHaveProperty("carBrandId");
+      expect(response.data.data[0]).toHaveProperty("title");
+    });
+
+    test("Get car models by id", async () => {
+      const response = await client.carController.getModelsByBrandId(
+        brands[0].id
+      );
+      expect(response.status, "Status code should be valid").toBe(200);
+      expect(response.data.data.length).toBeGreaterThan(0);
+      expect(response.data.data[0]).toHaveProperty("id");
+      expect(response.data.data[0]).toHaveProperty("carBrandId");
+      expect(response.data.data[0]).toHaveProperty("title");
+    });
+
+    test("Get user cars", async () => {
+      const response = await client.carController.getUserCars();
+      expect(response.status, "Status code should be valid").toBe(200);
+      expect(response.data.data.length).toBeGreaterThan(0);
+      expect(response.data.data[0]).toHaveProperty("id");
+      expect(response.data.data[0]).toHaveProperty("carBrandId");
+      expect(response.data.data[0]).toHaveProperty("carModelId");
+      expect(response.data.data[0]).toHaveProperty("mileage");
+    });
+
+    test("Get user cars by id", async () => {
+      const response = await client.carController.getUserCarsById(carId);
+      expect(response.status, "Status code should be valid").toBe(200);
+      expect(response.data.data).toHaveProperty("id");
+      expect(response.data.data).toHaveProperty("carBrandId");
+      expect(response.data.data).toHaveProperty("carModelId");
+      expect(response.data.data).toHaveProperty("initialMileage");
+      expect(response.data.data).toHaveProperty("updatedMileageAt");
+      expect(response.data.data).toHaveProperty("carCreatedAt");
+      expect(response.data.data).toHaveProperty("mileage");
+      expect(response.data.data).toHaveProperty("brand");
+      expect(response.data.data).toHaveProperty("model");
+      expect(response.data.data).toHaveProperty("logo");
```

Comment:
this test should use another user because it can be affected by other tests when you run your tests in parallel